salvo/src/salvo.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import site
+import sys
+
+# Run in the actual bazel directory so that the sys.path
+# is setup correctly
+if os.path.islink(sys.argv[0]):
```

Comment:
Can you help me understand this path manipulation and the `site.addsitedir("src")` call below?

I wouldn't expect this to be necessary in a bazel environment. We should be able to delete all these manipulations from here and the other files and just go with an import rooted at the beginning of the WORKSPACE:

`from src.lib.job_control_loader import load_control_doc`

I have tested this locally by removing all the manipulation and executing the `salvo` target which seems to have worked at least to a point that it imported all libraries correctly. Do we need to keep this or can we do without it?

Reply:
This is an artifact of us getting salvo to run in a cloud service.  We noticed that bazel would symlink files to the original location so, the current working directory would be the local source location and not `bazel-bin` where we expect.  This was true for the python generated code for the api.  We needed these files packaged in a library and were not able to accomplish that without the working directory manipulation.

I can remove it here, but there are other instance where we need it to remain.